### ISSUE TITLE
PRISM model syntax: C-style /* comments */

### ISSUE THREAD

**Issue by kleinj** [prismmodelchecker/prism#42]
Do we want to support /* C-style comments */?

**Comment by muhrik** [comment]
Is there a reason why not to support it?
As a beginner, I was disappointed to find out that there are no /* \*/ comments.
PRISM language has a plenty of features that make it look it is based on C, and lack of support for /* \*/ comments feels odd.
Actually, just yesterday I wanted to compare model checking results for two similar models and it would have been useful to copypaste both of them into PRISM and comment out the one I dont want to use.

**Comment by davexparker** [comment]
Yes, we should support this. It's actually very easy to add to the parser (via javacc). It's the syntax highlighting, e.g. in the GUI that needs some care, since it's done line by line.

**Comment by kleinj** [comment]
I had a brief look at the syntax highlighting code and indeed that seems to be a bit tricky. We'd probably need some additional global scanning that allows to determine, for each line, whether the beginning of that line is in a block comment. And additional logic to determine which lines are "damaged" (i.e., have to be redrawn) by a change of a block comment, as that's not line local anymore.

Highlighting for Latex / HTML export of the PRISM model is probably also affected.
